Makefile

```diff
@@ -52,7 +52,7 @@ kustomize: ## Installs kustomize in $PROJECT_DIR/bin
 
 ENVTEST = $(PROJECT_DIR)/bin/setup-envtest
 envtest: ## Installs setup-envtest in $PROJECT_DIR/bin
-	$(call go-get-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
+	$(call go-get-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.16)
```

Comment:
I should have sent a separate PR. It's failing in main.

It will also start failing in [kuadrant-operator](https://github.com/Kuadrant/kuadrant-operator/) BTW. https://github.com/kubernetes-sigs/controller-runtime/issues/2720

Reply:
Weird, as we do not use envtest in kuadrant-operator

Reply:
> Weird, as we do not use envtest in kuadrant-operator

Never mind then.